client-ui/src/app/model/property.ts

```diff
@@ -28,11 +28,13 @@ export class Property {
 
 export const enum PropertyType {
     Join = "Join",
+    Alias = "Alias",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Strings must use singlequote. (quotes)](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4585293)

Reply:
Do we use double quotes everywhere? Should I set this rule to check for double quotes instead?

Reply:
Well I am trying to use double quote everywhere ye